lib/Board.mjs

```diff
@@ -258,7 +258,7 @@ export default class Board {
         return pieceValue
     }
 
-    getMoves (color = this.getPlayingColor(), movablePiecesRequiredToSkipTest = null) {
+    getMoves (color = this.getPlayingColor(), movablePiecesRequiredToSkipTest = null, ignoreCheckValidation = false) {
```

Comment:
As there might be more custom options, I would move it to some options object, WDYT?